.github/workflows/ci.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - run: yarn compile
+      - run: yarn aio:compile
```

Comment:
A lot of this config is here because our current repo architecture doesn't allow for individual packages to have different config. I imagine what would replace this is something like `turborepo` or `nx` where each individual package is responsible for it's own config and these task runners can mimic the current behaviour.